packages/theming/apple-theme/src/appleTheme.colors.macos.ts

```diff
@@ -0,0 +1,223 @@
+import { AppleSemanticPalette } from './appleSemanticColors.macos';
+
+/** creates a palette of colors for the apple theme, given the FluentUI Apple Palette and Apple Semantic Palette */
+export function paletteFromAppleColors(fluentApple: ApplePalette, appleSemantic: AppleSemanticPalette): ThemeColorDefinition {
+  return {
+    /* PaletteBackgroundColors & PaletteTextColors */
+
+    background: appleSemantic.windowBackgroundColor,
+    bodyStandoutBackground: appleSemantic.underPageBackgroundColor,
+    bodyFrameBackground: appleSemantic.windowBackgroundColor,
+    bodyFrameDivider: appleSemantic.separatorColor,
+    bodyText: appleSemantic.textColor,
+    bodyTextChecked: appleSemantic.selectedTextColor,
+    subText: appleSemantic.placeholderTextColor,
+    bodyDivider: appleSemantic.separatorColor,
+
+    // TODO
+    disabledBackground: fluentApple.gray100,
+    disabledText: appleSemantic.tertiaryLabelColor,
+    disabledBodyText: appleSemantic.tertiaryLabelColor,
+    disabledSubtext: appleSemantic.quaternaryLabelColor,
+    disabledBodySubtext: appleSemantic.quaternaryLabelColor,
+
+    focusBorder: 'transparent',
+    variantBorder: appleSemantic.separatorColor,
+    variantBorderHovered: appleSemantic.separatorColor,
+    defaultStateBackground: appleSemantic.controlBackgroundColor,
+
+    // TODO
+    errorText: fluentApple.dangerPrimary,
+    warningText: fluentApple.warningPrimary,
+    errorBackground: fluentApple.dangerTint10,
+    blockingBackground: fluentApple.dangerTint10,
+    warningBackground: fluentApple.warningPrimary,
+    warningHighlight: fluentApple.warningTint10,
+    successBackground: fluentApple.successTint10,
+
+    inputBorder: appleSemantic.separatorColor,
+    inputBorderHovered: appleSemantic.separatorColor,
+    inputBackground: appleSemantic.textBackgroundColor,
+    inputBackgroundChecked: appleSemantic.selectedContentBackgroundColor,
+    inputBackgroundCheckedHovered: appleSemantic.selectedContentBackgroundColor,
+    inputForegroundChecked: fluentApple.communicationBlue,
+    inputFocusBorderAlt: appleSemantic.keyboardFocusIndicatorColor,
+    smallInputBorder: appleSemantic.separatorColor,
+    inputText: appleSemantic.textColor,
+    inputTextHovered: appleSemantic.textColor,
+    inputPlaceholderText: appleSemantic.placeholderTextColor,
+
+    buttonBackgroundChecked: fluentApple.communicationBlueTint10,
+    buttonBackgroundHovered: fluentApple.communicationBlue,
+    buttonBackgroundCheckedHovered: fluentApple.communicationBlueTint10,
+    buttonBackgroundPressed: fluentApple.communicationBlueTint10,
+    buttonBackgroundDisabled: fluentApple.gray100,
+    buttonText: appleSemantic.controlTextColor,
+    buttonTextHovered: appleSemantic.controlTextColor,
+    buttonTextChecked: appleSemantic.controlTextColor,
+    buttonTextCheckedHovered: appleSemantic.controlTextColor,
+    buttonTextPressed: appleSemantic.controlTextColor,
+    buttonTextDisabled: appleSemantic.controlTextColor,
```

Comment:
@chiuam The JS button text is themed with tokens from here, along with similar tokens in the ControlColorTokens section